products/sle/main.pm

```diff
@@ -918,6 +918,20 @@ elsif (get_var("VIRT_AUTOTEST")) {
         loadtest "virt_autotest/guest_migration_dst";
     }
 }
+elsif (get_var("PERF_KERNEL")) {
```

Comment:
May I kindly ask you to update `variables.md` with proper explanation what these new variables do, including ones you have in other places in the code (e.g. `SPECIFIC_DISK`).

Reply:
Updated variables.md .